packages/server/src/auth/local.strategy.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy } from 'passport-local';
+import { AuthService } from './auth.service';
+
+@Injectable()
+export class LocalStrategyUsername extends PassportStrategy(Strategy, 'username') {
+	constructor(private authService: AuthService) {
```

Comment:
NIT: It looks like the new files are using tabs when the rest of the project seems to be using spaces

Reply:
Do we have a standard in place? Not sure if we should use spaces or tabs.

Reply:
Yea, there is a top level `.editorconfig` file which can be used. Depending on your IDE there are plugins to automatically follow the format specified in it